### ISSUE TITLE
Variable quoting required to meet stricter syntax in ansible 2.0

### ISSUE THREAD

**Issue by UTAS-Nick-Howden** [awailly/cis-ubuntu-ansible#121]
In file tasks/section_08_level2.yml

with_items: audit_lines_for_find.stdout_lines
should be 
with_items: "{{ audit_lines_for_find.stdout_lines }}"

Without the extra quoting the generated file contains the line
audit_lines_for_find.stdout_lines
which is not the intended behaviour

Believe this is because of stricter syntax in Ansible 2.0

**Comment by awailly** [comment]
Yes, nice catch. Do you mind creating the pull request to be sure that you have the correct fix?

**Comment by eastokes** [comment]
I've noticed several other variables and commands that should be quoted to follow best practices. I'll likely take on this work.
